docs/cow-protocol/tutorials/cow-swap/native.mdx

```diff
@@ -86,3 +86,15 @@ Fear not! If the order fails, your ETH can be refunded into your account.
 The refund can be triggered by any account and should happen automatically if you used the CoW Swap UI. If not, get in touch with us for help at [info@cow.fi](mailto:info@cow.fi)!
 
 :::
+
+:::note
+
+The [Eth-flow](/cow-protocol/reference/contracts/periphery/eth-flow) option is NOT available for Smart Contract wallets.
```

Comment:
```suggestion
The [Eth-flow](/cow-protocol/reference/contracts/periphery/eth-flow) option is NOT available for Smart Contract wallets **in CoW Swap**.
```

While the context is related to the CoW Swap tutorials, an explicit statement with a carve out here I think is good as `Eth-Flow` can certainly be used by a smart contract.